azure-mgmt-redis/src/main/java/com/microsoft/azure/management/redis/implementation/RedisAccessKeysImpl.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.redis.implementation;
+
+import com.microsoft.azure.management.redis.RedisAccessKeys;
+import com.microsoft.azure.management.redis.RedisCache;
+
+/**
+ * The {@link RedisCache#keys} action result.
+ */
+public class RedisAccessKeysImpl implements RedisAccessKeys {
```

Comment:
why is this impl class public?  in general, our Impl classes are package-internal (i.e. no `public` keyword)